tests/test_pipeline_inference_only.py

```diff
@@ -4,16 +4,18 @@
 import jax.numpy as jnp
 
 from alpa import init, PipeshardParallel
-from alpa.testing import (MLPModel, create_train_state, get_mlp_inference_step,
-                          assert_allclose)
+from alpa.model.bert_model import BertConfig, FlaxBertLayerCollection
+from alpa.testing import (MLPModel, create_train_state,
+                          get_bert_layer_collection_inference_step,
+                          get_mlp_inference_step, assert_allclose)
 
 
 class PipelineInferenceTest(unittest.TestCase):
+
     def setUp(self):
         init(cluster="ray")
```

Comment:
If we do not call `alpa.shutdown()`, the second test will follow the physical mesh group created by the first test. Is this intended?

Reply:
it's a mistake